modules/olm-filtering-fbc.adoc

```diff
@@ -0,0 +1,149 @@
+$ opm render <registry>/<namespace>/<catalog_image_name>:<tag> \
+    -o yaml > <catalog_dir>/index.yaml
+----
++
+[NOTE]
+====
+Alternatively, you can use the `-o json` flag to output in JSON format.
+====
+
+. Modify the contents of the resulting `index.yaml` file to remove one or more Operator packages.
++
+[IMPORTANT]
+====
+After a bundle has been published in a catalog, assume that one of your users has installed it. Therefore, avoid stranding users that have that version installed by making sure that all previously published bundles in a catalog have an update path to the current or newer channel head.
+====
++
+The following example shows a set of `olm.package`, `olm.channel`, and `olm.bundle` blobs that represent an Operator package to remove from the file:
++
+.Example removed entries
+[%collapsible]
+====
+[source,yaml]
+----
+---
+defaultChannel: release-2.7
+icon:
+  base64data: <base64_string>
+  mediatype: image/svg+xml
+name: example-operator
+schema: olm.package
+---
+entries:
+- name: example-operator.v2.7.0
+  skipRange: '>=2.6.0 <2.7.0'
+- name: example-operator.v2.7.1
+  replaces: example-operator.v2.7.0
+  skipRange: '>=2.6.0 <2.7.1'
+- name: example-operator.v2.7.2
+  replaces: example-operator.v2.7.1
+  skipRange: '>=2.6.0 <2.7.2'
+- name: example-operator.v2.7.3
+  replaces: example-operator.v2.7.2
+  skipRange: '>=2.6.0 <2.7.3'
+- name: example-operator.v2.7.4
+  replaces: example-operator.v2.7.3
+  skipRange: '>=2.6.0 <2.7.4'
+name: release-2.7
+package: example-operator
+schema: olm.channel
+---
+image: example.com/example-inc/example-operator-bundle@sha256:<digest>
+name: example-operator.v2.7.0
+package: example-operator
+properties:
+- type: olm.gvk
+  value:
+    group: example-group.example.io
+    kind: MyObject
+    version: v1alpha1
+- type: olm.gvk
+  value:
+    group: example-group.example.io
+    kind: MyOtherObject
+    version: v1beta1
+- type: olm.package
+  value:
+    packageName: example-operator
+    version: 2.7.0
+- type: olm.bundle.object
+  value:
+    data: <base64_string>
+- type: olm.bundle.object
+  value:
+    data: <base64_string>
+relatedImages:
+- image: example.com/example-inc/example-related-image@sha256:<digest>
+  name: example-related-image
+schema: olm.bundle
+---
+----
+====
+
+. Save your changes to the `index.yaml` file.
+
+. Validate the catalog:
++
+[source,terminal]
+----
+$ opm validate <catalog_dir>
+----
+
+. Rebuild the catalog:
++
+[source,terminal]
+----
+$ podman build . \
+    -f <catalog_dir>.Dockerfile \
+    -t <registry>/<namespace>/<catalog_image_name>:<tag>
+----
+
+. Push the updated catalog image to a registry:
++
+[source,terminal]
+----
+$ podman push <registry>/<namespace>/<catalog_image_name>:<tag>
+----
+
+.Verification
+
+. In the web console, navigate to the OperatorHub configuration resource in the *Administration* -> *Cluster Settings* -> *Configuration* page and update or add the catalog source to use the pull spec for your updated catalog image.
```

Comment:
"update or add the catalog source"
Does this need add'l reference, or is it well-understood by the audience. 

nit:  I find the interlaced clauses to be a bit more challenging when reading, and prefer something like
```and add the catalog source or update the existing catalog source to use the pull spec for your updated catalog image.
```